### ISSUE TITLE
musl in configuration section.

### ISSUE THREAD

**Issue by Duncaen** [void-linux/void-docs#267]
* I feel like this is more relevant to the installation section, there is not a single thing that would be related to configuring already installed systems.
* Examples on how to create void linux chroots don't really belong here, I think it would make sense to add a chroot page to the installation section and link from the musl page to it.

**Comment by flexibeast** [comment]
There's currently a PR for a chroot installation guide: https://github.com/void-linux/void-docs/pull/177. Once that's finalised, @ericonr is planning to work on further installation-related content that leverages that page.

That PR also has a [discussion](https://github.com/void-linux/void-docs/pull/177#pullrequestreview-403985709) about the appropriate `mount` commands; istr someone saying that setting up a glibc chroot on an existing musl system is a different situation than setting up a chroot for installation. If that is indeed the case, then it seems to me that the chroot content could maybe be retained; if not, then it could be removed.

At any rate, we can certainly move this page to the "Installation" section.

**Comment by Duncaen** [comment]
I was thinking more about a "How to setup Void Linux container/chroot rootfs", which I think is useful and quite different from installing void linux on bare metal using a chroot.

**Comment by travankor** [comment]
Some musl-related things that could be documented:

- gcompat
- utmps (utmp/wtmp for musl)
- libexecinfo (backtraces)
- [musl-locales](https://gitlab.com/rilian-la-te/musl-locales)

**Comment by flexibeast** [comment]
@travankor:

* re. `gcompat`, it wasn't included in the current `musl` page due to @q66's comment [here](https://github.com/void-linux/void-docs/pull/163#discussion_r395958344).
* re. `utmps`, it's not currently packaged, but i've starting putting together a package for it, though i've not yet submitted a PR - if i created a WIP PR, would you be able to help with that?

**Comment by travankor** [comment]
@flexibeast Yes, I can help package utmps. I'm a little busy in the next 2-3 days, though, so maybe wait a few days.

**Comment by flexibeast** [comment]
@travankor: Thanks! i've just opened a WIP PR, https://github.com/void-linux/void-packages/pull/21869, but certainly no rush, as i've got lots of other things on my plate myself. :-)

**Comment by ericonr** [comment]
@flexibeast regarding gcompat for musl, I know of at least one case where it works perfectly. The toolchain that `arduino-cli` downloads is linked to glibc but works in a musl system as long as you have gcompat. I recently added it as a dependency, even. Therefore, I believe it might make sense to mention it, because it can work.

**Comment by flexibeast** [comment]
@ericonr: \*nod\* i'm certainly happy to mention it, myself, but i deferred to @q66 as someone with much more knowledge and experience (particularly with Void) than me .... So basically @q66 is the person you need to convince. :-)

**Comment by flexibeast** [comment]
Also, what do @void-linux/pkg-committers think of packaging `musl-locales` and referencing it in the Handbook?

**Comment by Duncaen** [comment]
This also shouldn't mention proot, using it on travis has proven that its really not a good idea and just results in random crashes.
Mentioning something like bubblewrap would be more appropriate.
